web/index.ts

```diff
@@ -166,6 +167,23 @@ createConnection().then(() => {
         app.use("/api/screening", screenerApiRouter);
     }
 
+    function configureParticipationCertificateAPI() {
+        const participationCertificateRouter = express.Router();
+        participationCertificateRouter.get("/:certificateId", (req, res, next) => {
+            if(!req.subdomains.includes("verify")){
+                return next();
+            }
+            certificateController.confirmCertificateHandler(req, res)
+        });
+        participationCertificateRouter.use((req, res, next) =>{
+            if(req.subdomains.includes("verify")){
+                return res.status(404).end()
+            }
+            next();
+        });

```

Comment:
Catching the `verify.` subdomain this way will work – but this will lead to some errors. For example if a user goes to https://verify.corona-school.de/1DC0F8AA10 then the browser will usually also request https://verify.corona-school.de/favicon.ico (which isn't handled appropriately). This will then log an error every time, which should be avoided.

Reply:
Added favicon to assets [https://github.com/corona-school/backend/commit/445c87302c6acb68e8a0ee0aa7db222242263dcb](https://github.com/corona-school/backend/commit/445c87302c6acb68e8a0ee0aa7db222242263dcb) -- any other concerns?